### PR TITLE
[12.0] [MIG] stock_barcodes_gs1

### PR DESCRIPTION
**Pull request by JoanSForgeFlow**
Standard stock_barcodes_gs1 migration to v12.

Depends on #235 

cc ~ @ForgeFlow @LoisRForgeFlow

### CODE REVIEW

**Comment by LoisRForgeFlow** [comment]
@JoanSForgeFlow can you check travis? tests are failling.

File: stock_barcodes_gs1/__manifest__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 {
     "name": "Stock Barcodes GS1",
     "summary": "It provides read GS1 barcode on stock operations.",
-    "version": "11.0.1.0.1",
+    "version": "12.0.1.0.1",
```
Comment (LoisRForgeFlow) [inline stock_barcodes_gs1/__manifest__.py]: 12.0.1.0.0

**Review by rousseldenis — APPROVED** [review]

**Review by LoisRForgeFlow — APPROVED** [review]
:+1:

**Pull request merged**